speech_assets/SampleUtterances.txt

```diff
@@ -1,3 +1,5 @@
+AMAZON.HelpIntent help
+AMAZON.HelpIntent open help
```

Comment:
You might not know, but you shouldn't need to add samples for the built in intents unless you wanted to extend the built in list of sample phrases with something specific.

It does make things easier as you don't have to provide translations for current and future language support. These standard built-in intents are already supported the three 

https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/built-in-intent-ref/standard-intents#available-standard-built-in-intents